modules/govuk/manifests/apps/rummager.pp

```diff
@@ -98,10 +98,6 @@
     process_type   => 'publishing-api-document-indexer',
   }
 
-  govuk_rabbitmq::monitor_consumers {'rummager-publishing-api-document-indexer':
```

Comment:
I think we still want to monitor the consumer.  Was this meant to have been moved into the `rummager/rabbitmq.pp` file?